### ISSUE TITLE
Implement conversations

### ISSUE THREAD

**Issue by rbeezer** [PreTeXtBook/pretext#1801]
Some textbooks like to have a dialogue between teacher and student, much like a script or screenplay, with named participants.  An example is here: https://gttp.info/q08e9fb0n.html which is a draft and could expire/move.  "Getting to the Point, Daniel Goroff and Nat Kuhn"

Some overall elment, `conversation`, dialogue`?  Structured by `speaker`, `participant`, etc?

I recall a discrete math textbook now in PreTeXt that has something like this?  Maybe Keller and Trotter?

**Comment by kcrisman** [comment]
>
> Some textbooks like to have a dialogue between teacher and student, much
> like a script or screenplay, with named participants. An example is here:
> https://gttp.info/q08e9fb0n.html which is a draft and could expire/move.
> "Getting to the Point, Daniel Goroff and Nat Kuhn"
>
> I've seen this too. Idea: Should this (since there is already the poetry
and music stuff for the arts) be implemented as part of something extending
PTX capabilities to scripts, and then allowed in the appropriate containers
for the "pedagogical" version to be allowed in addition to (say) Acts or
Scenes?

**Comment by rbeezer** [comment]
> implemented as part of something extending

Yes, I'd like to do something general enough that it could be seamlessly extended this way.  But I have almost no practical experience, so we'd want to find some help from somebody who does.

Thanks for adding this to the issue!

Rob
